pkgs/top-level/qt5-packages.nix

```diff
@@ -235,6 +235,7 @@ in (kdeFrameworks // plasmaMobileGear // plasma5 // plasma5.thirdParty // kdeGea
   # Not a library, but we do want it to be built for every qt version there
   # is, to allow users to choose the right build if needed.
   sddm = callPackage ../applications/display-managers/sddm { };
+  sddm-catppuccin = callPackage ../applications/display-managers/sddm/sddm-catppuccin.nix {};
```

Comment:
In qt packages only offficial qt things or libraries are meant to be put in. The theme should just live in top-level.

Reply:
it has been moved to the top-level and renamed to fit in with the other catppuccin themes.